app/Providers/AppServiceProvider.php

```diff
@@ -23,6 +24,6 @@ public function register()
      */
     public function boot()
     {
-        //
+        Paginator::defaultView('vendor.pagination.bootstrap-4');
```

Comment:
у нас вроде 5 bootstrap (некритично)
```suggestion
        Paginator::defaultView('vendor.pagination.bootstrap-5');
```

Давай удалим лишнее шаблоны в resources/views/vendor/pagination
оставь только нужный